### PR TITLE
:sparkles: Add fiberi18n middleware

### PR DESCRIPTION
**Pull request by Skyenought**
Add middleware to [go-i18n](https://github.com/nicksnyder/go-i18n) support for Fiber.

### Quick Start
```go
func main() {
	app := fiber.New()
	app.Use(
		fiberi18n.New(&fiberi18n.Config{
			RootPath:        "./localize",
			AcceptLanguages: []language.Tag{language.Chinese, language.English},
			DefaultLanguage: language.Chinese,
		}),
	)
	app.Get("/", func(c *fiber.Ctx) error {
		return c.SendString(fiberi18n.MustGetMessage("welcome"))
	})
	app.Get("/:name", func(ctx *fiber.Ctx) error {
		return ctx.SendString(fiberi18n.MustGetMessage(&i18n.LocalizeConfig{
			MessageID: "welcomeWithName",
			TemplateData: map[string]string{
				"name": ctx.Params("name"),
			},
		}))
	})
	app.Listen("127.0.0.1:3000")
}

```

### CODE REVIEW

**Review by gaby — CHANGES_REQUESTED** [review]
Can you add an entry in .github/dependabot.yml for this middleware?

**Comment by Skyenought** [comment]
> Can you add an entry in .github/dependabot.yml for this middleware?

ok

**Comment by ReneWerner87** [comment]
@Skyenought

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
Is there a reason why you have not updated the security yaml ? have you only read half the comment ? 

https://github.com/gofiber/contrib/blob/main/.github/workflows/security.yml

**Comment by Skyenought** [comment]
Sorry!

**Comment by gaby** [comment]
@Skyenought Can you update the go.mod to 1.18 and run `go mod tidy`

**Comment by Skyenought** [comment]
> @Skyenought Can you update the go.mod to 1.18 and run `go mod tidy`

ok

**Comment by gaby** [comment]
@Skyenought For some reason its pointing to fiber 2.39.x, latest should be 2.40.1 l. May need to run `go get -u ./...`

**Review by ReneWerner87 — APPROVED** [review]

**Comment by ReneWerner87** [comment]
@Skyenought pls check 
image

**Comment by Skyenought** [comment]
Strangely enough, I couldn't figure out what went wrong

**Comment by ReneWerner87** [comment]
@Skyenought any progress ?

**Comment by Skyenought** [comment]
I recently got COVID-19, so not much progress, sorry.

**Comment by gaby** [comment]
@Skyenought Can you run `go test -cover ./...`, what's the current coverage % for this middleware?

**Comment by Skyenought** [comment]
> go test -cover ./...

ok      github.com/gofiber/contrib/fiberi18n    0.684s  coverage: 77.5% of statements
?       github.com/gofiber/contrib/fiberi18n/example    [no test files]

**Comment by ReneWerner87** [comment]
@Skyenought can you please update the middleware and check the workflows, when the workflows are running and the documentation is there i will merge it right away

**Comment by Skyenought** [comment]
> @Skyenought can you please update the middleware and check the workflows, when the workflows are running and the documentation is there i will merge it right away

Get

**Comment by ReneWerner87** [comment]
@Skyenought i think the changes in the root readme.md and the release drafter workflow+config files are missing

**Review by efectn — APPROVED** [review]

**Review by gaby — APPROVED** [review]
LGTM

File: fiberi18n/go.sum [hunk 1]
```diff
@@ -0,0 +1,79 @@
+github.com/BurntSushi/toml v1.0.0 h1:dtDWrepsVPfW9H/4y7dDgFc2MBUSeJhlaDtK13CxFlU=
```
[the remark this answers is not in the document]
Comment (Skyenought) [reply fiberi18n/go.sum:1]: GET

**Comment by gaby** [comment]
@ReneWerner87 I think this is good to merge now.

**Comment by ReneWerner87** [comment]
Ok will merge it tomorrow morning

**Comment by Skyenought** [comment]
@ReneWerner87 Thanks!

**Comment by ReneWerner87** [comment]
https://github.com/gofiber/contrib/releases/tag/fiberi18n%2Fv0.1.0 
thx for the effort

**Pull request merged**